src/sagemaker_containers/serializers.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the 'license' file accompanying this file. This file is
+# distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import json as stdjson
```

Comment:
we should not rename this, and wouldn't have to if we didn't declare a module level json object of our own

Reply:
Can you point me to the PEP, lint, or code standard that says to not do that?

Reply:
it's confusing... people reading are likely to know what the 'json' lib does, and won't know what stdjson is unless they go back and look at imports.

Reply:
Looking at the imports is just a normal thing to do. Do you have any suggestion besides the unnecessary big statement  serializer.JsonSerializer().dump ? Remember that because of things like that, nobody likes Java.

Reply:
this is unrelated to the package/class name issue. the google style guide specifically says not to use `import ... as` to rename a package unless it's "inconveniently long" which `json` definitely isn't. the only reason we "need" the rename is to avoid collision with the module level variable name json, which the google style also discourages.